Add that agents are namespaced (#1108)
sensu/sensu-docs: content/sensu-go/5.0/reference/agent.md

| namespace |      |
---------------|------

> description    | Agent namespace _NOTE: Agents can only have one namespace._

Reviewer: ```suggestion
description    | Agent namespace _NOTE: Agents can only belong to a single namespace._
```

Reviewer: Perhaps we should go a little further to say `Agents are represented in the backend a class of entity. Entities can only belong to a single namespace.` with a link to the table of namespaced resources
